localgovdrupal/localgov_microsites_group
# Fix schema error #492

## What does this change?

Removes invalid schema from group media view.

Adds GitHub shared workflow to fix test runs.

## How to test

See the tests pass. Check that the site media listing still works after an install.
